assets/css/style.css

```diff
@@ -396,7 +396,12 @@ span.morethan90 {
 
 .editor .strings {
 	float: left;
-	width: 60%;
+	min-width: 60%;
+}
+
+.editor .strings .textareas textarea {
+	width: 95%;
+	resize: none;
```

Comment:
This should be `resize: vertical` to allow users to resize the textarea vertically if a string has more than ~7 lines.